dev/ci/integration/setup-deps.sh

```diff
@@ -2,8 +2,8 @@
 
 set -euo pipefail
 
-yarn
-yarn generate
+pnpm install
```

Comment:
Shouldn't CI scripts run `install` with `--frozen-lockfile`?

```suggestion
pnpm install --frozen-lockfile
```